src/perf/lib/PerfServer.cpp

```diff
@@ -134,8 +134,8 @@ PerfServer::ConnectionCallback(
             if (PrintStats) {
                 QuicPrintConnectionStatistics(MsQuic, ConnectionHandle);
             }
-            MsQuic->ConnectionClose(ConnectionHandle);
         }
+        MsQuic->Release(ConnectionHandle);
```

Comment:
Why did you move this?

Reply:
Because we always want to call this now. AppCloseInProgram can technically never happen now (Which means I need to move the rest out).